leaderboard/database/redis.go

```diff
@@ -15,6 +15,7 @@ type Redis struct {
 
 var _ Database = &Redis{}
 
+// ExpirationSet is used to list expirations set that worker will use to remove members
 const ExpirationSet string = "expiration-sets"
```

Comment:
It seems this is not used anywhere in this PR, maybe it should not be here?

Reply:
This was already used by `SetMembersTTL` function. Here I am only adding the comment https://github.com/topfreegames/podium/blob/025e6b4dcf77892bc1ba2e5c51e784c9ace0b403/leaderboard/database/redis.go#L283